We don't need this as it is for compatibility with previous
versions of OCaml, so avoid adding an unnecessary dependency.

--- a/src/dune
+++ b/src/dune
@@ -4,5 +4,4 @@
  (wrapped false)
  (install_c_headers ocaml_integers)
  (c_names unsigned_stubs)
- (libraries stdlib-shims)
  (synopsis "Signed and unsigned integers of various sizes"))
